RELEASES.md

```diff
@@ -1,3 +1,80 @@
+Version 1.42.0 (2020-03-12)
+==========================
+
+Language
+--------
+- [You can now use `#[repr(transparent)]` on univariant `enum`s.][68122] Meaning
+  that you can create an enum that has the exact layout and ABI of the type
+  it contains.
+- [You can now use the slice pattern syntax with subslices.][67712] e.g.
+  ```rust
+  fn foo(words: &[&str]) {
+      match words {
+          ["Hello", "World", "!", ..] => println!("Hello World!"),
+          ["Foo", "Bar", ..] => println!("Baz"),
+          rest => println!("{}", rest),
+      }
+  }
+  ```
+  <!-- TODO -->
+- [Merge `TraitItem` & `ImplItem into `AssocItem`][67131]
+
+Compiler
+--------
+- [Added tier 3\* support for `armv7a-none-eabi`.][68253]
+- [Added tier 3 support for `riscv64gc-unknown-linux-gnu`.][68037]
+- [`Option::{expect,unwrap}` and
+   `Result::{expect, expect_err, unwrap, unwrap_err}` now produce panic messages
+   pointing to the location where they were called, rather than
+   `core`'s internals. ][67887]
+
+Libraries
+---------
+- [`iter::Empty<T>` now implements Send and Sync for any `T`.][68348]
+- [`Pin::{map_unchecked, map_unchecked_mut}` no longer require the return type
+   to implement `Sized`.][67935]
+- [`io::Cursor` now derives `PartialEq` and `Eq`.][67233]
+- [`Layout::new` is now `const`.][66254]
+- [Added Standard Library support for `riscv64gc-unknown-linux-gnu`.][66899]
+
+
+Stabilized APIs
+---------------
+- [`DebugMap::key`]
+- [`DebugMap::value`]
+- [`ManuallyDrop::take`]
+- [`matches!`]
+- [`ptr::slice_from_raw_parts_mut`]
+- [`ptr::slice_from_raw_parts`]
+- [`wait_timeout_while`]
```

Comment:
Link to https://doc.rust-lang.org/std/sync/struct.Condvar.html#method.wait_timeout_while is missing

```suggestion
- [`Condvar::wait_timeout_while`]
```